03.bowling/bowling.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+
+scores = score.split(',')
+
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = []
+shots.each_slice(2) do |shot|
+  frames << shot
+end
+
+point = 0
+frames.each.with_index do |frame, i|
```

Comment:
```suggestion
point = frames.each_with_index.sum do |frame, i|
```

のようにsumの結果を直接変数に入れる方法も採れそうです。

Reply:
こちら、ありがとうございます🙇‍♂️
こちらのコードで動くことを確認しました。
ただ、ここの理解が怪しいため、フィヨルドの提出ページの方で疑問点を質問させていただいております。
そちらをご確認いただけるとありがたいです。宜しくお願いいたします🙏